checkov/common/bridgecrew/platform_integration.py

```diff
@@ -141,7 +139,8 @@ def setup_http_manager(self, ca_certificate=os.getenv('BC_CA_BUNDLE', None)):
                 self.http = urllib3.PoolManager()
 
     def setup_bridgecrew_credentials(self, repo_id, skip_fixes=False, skip_suppressions=False,
-                                     skip_policy_download=False, source=None, source_version=None, repo_branch=None):
+                                     skip_policy_download=False, source=None, source_version=None, repo_branch=None,
+                                     prisma_api_url=os.getenv('PRISMA_API_URL', None)):
```

Comment:
```suggestion
                                     prisma_api_url):
```

I don't think we need the default value here, because it will just get called from main.py using the config value.

Reply:
Done.